docs/basics/networkvariable.md

```diff
@@ -254,7 +260,7 @@ There are two options for reading a `NetworkVariable.Value`:
 
 There are two options for writing a `NetworkVariable.Value`:
 
-- **Server(default)**: the server is the only one that can write the value.
+- **Server (default)**: the server is the only one that can write the value.
```

Comment:
We might make a note here that when using a distributed authority network topology the write permissions will automatically be updated to owner write permissions (even if they have set server write permissions). This automatic conversion is just to help covert a project started with a client-server network topology (i.e. conversion) and/or for projects that want to support both client-server and distributed authority network topologies.